src/scrape/scrape_job_test.go

```diff
@@ -144,18 +145,50 @@ var kubeletSpecs = definition.SpecGroups{
 	"container": metric.KubeletSpecs["container"],
 }
 
-type testGrouper struct{}
+type testGrouper struct {
```

Comment:
I'm probably complaining about code that was already here (sorry!), but I'm finding hard to understand what `testGrouper` is. Is it a helper? A mock? What does it do? I'd love to see a godoc comment explaining it :)

Reply:
I just extended the mock we had there... Anyway You are right it was not obvious and now which it does more stuff it may confusing. I'll improve naming and maybe include some godocs

Reply:
Yep, I don't think it needs to be exhaustive since it's unexported and for tests. Oneliners such as `// testGrouper is an implementation of the Grouper interface that returns mocked metrics that might change in subsequent calls.` will be more than enough :)